interface/patient_file/summary/demographics.php

```diff
@@ -473,120 +476,205 @@ function setMyPatient() {
 </head>
 
 <body class="body_top patient-demographics">
-
```

Comment:
Need to separate meaningful code changes from the formatting improvement or else we can't review the meaningful code changes.